src/depthai_wrapper.cpp

```diff
@@ -0,0 +1,169 @@
+#include "depthai/depthai_wrapper.hpp"
+
+namespace DepthAI
+{
+
+DepthAI::DepthAI(std::string usb_device, std::string config_file, bool usb2_mode) :
+      Device(usb_device, usb2_mode)
+{
+
+    std::ifstream file(config_file);
+    std::ostringstream file_stream;
+    file_stream << file.rdbuf();
+    std::string config_str = file_stream.str();
+
+    pipeline_ = this->create_pipeline(config_str);
+    config_json_ = nlohmann::json::parse(config_str);
+
+    this->set_resolution();
+    this->create_frame_holders();
+
+    width_map_[720] = 1280;
+    width_map_[800] = 1280;
+    width_map_[400] = 640;
+}
```

Comment:
I think you should add a helpful print here for the final sizes chosen for the resolutions

Reply:
Looks like you missed this

Reply:
it is not there. I am not sure why it is not showing here.

Reply:
I don't understand what you mean. I said, maybe you should add a print here, at the end of the constructor, stating what width, height and streams were chosen.

Reply:
Streams chosen were the ones they specified. and it is being written into the terminal by them which all is started so I don't think we need to do that. but for resolution I will add

Reply:
oh okay sounds good :)